lib/ansible/modules/cloud/amazon/cloudfront_distribution.py

```diff
@@ -687,272 +687,272 @@
+              description: Whether the query string is used in cache lookups.
               returned: always
               type: bool
               sample: false
             query_string_cache_keys:
-              description: Which query string keys to use in cache lookups
+              description: Which query string keys to use in cache lookups.
               returned: always
               type: complex
               contains:
                 quantity:
-                  description: Count of query string cache keys to use in cache lookups
+                  description: Count of query string cache keys to use in cache lookups.
                   returned: always
                   type: int
                   sample: 1
                 items:
-                  description: List of query string cache keys to use in cache lookups
+                  description: List of query string cache keys to use in cache lookups.
                   returned: when list is not empty
                   type: list
                   sample:
         lambda_function_associations:
-          description: Lambda function associations for a cache behavior
+          description: Lambda function associations for a cache behavior.
           returned: always
           type: complex
           contains:
             quantity:
-              description: Count of lambda function associations
+              description: Count of lambda function associations.
               returned: always
               type: int
               sample: 1
             items:
-              description: List of lambda function associations
+              description: List of lambda function associations.
               returned: when list is not empty
               type: list
               sample:
               - lambda_function_arn: arn:aws:lambda:123456789012:us-east-1/lambda/lambda-function
                 event_type: viewer-response
         max_ttl:
-          description: Maximum Time to Live
+          description: Maximum Time to Live.
           returned: always
           type: int
           sample: 31536000
         min_ttl:
-          description: Minimum Time to Live
+          description: Minimum Time to Live.
           returned: always
           type: int
           sample: 0
         path_pattern:
-          description: Path pattern that determines this cache behavior
+          description: Path pattern that determines this cache behavior.
           returned: always
           type: str
           sample: /path/to/files/*
         smooth_streaming:
-          description: Whether smooth streaming is enabled
+          description: Whether smooth streaming is enabled.
           returned: always
           type: bool
           sample: false
         target_origin_id:
-          description: Id of origin reference by this cache behavior
+          description: Id of origin reference by this cache behavior.
```

Comment:
```suggestion
          description: ID of origin reference by this cache behavior.
```
would you mind fixing this up while we're in here?